openspec/changes/archive/2026-06-05-d08-auth-user-consolidation/design.md

```
@@ -0,0 +1,67 @@
+## Context
+
+The `AuthUser` interface exists in three places in the codebase:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)